### PR TITLE
Upgrade PHPUnit and dependencies.

### PR DESCRIPTION
**Pull request by markstory**
Prepare for the 1.0.0 release.

Refs cakephp/app#502

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     ],
     "require-dev": {
         "composer/composer": "1.0.*@dev",
+        "phpunit/phpunit": "^5.7|^6.0",
```
Comment (antograssiot) [inline composer.json]: PHP 5.5 will require PHPUnit 4.8 so I guess you should use `^4.8|^5.7|^6.0`

Comment (markstory) [inline composer.json]: Thanks!

**Comment by ADmad** [comment]
LGTM

**Comment by markstory** [comment]
Thanks for the help on this @antograssiot

**Pull request merged**